examples/specialty_plots/anscombe.py

```diff
@@ -20,30 +22,35 @@
 x4 = [8, 8, 8, 8, 8, 8, 8, 19, 8, 8, 8]
 y4 = [6.58, 5.76, 7.71, 8.84, 8.47, 7.04, 5.25, 12.50, 5.56, 7.91, 6.89]
 
+datasets = {
+    'I': (x, y1),
+    'II': (x, y2),
+    'III': (x, y3),
+    'IV': (x4, y4)
+}
+
 
 def fit(x):
     return 3 + 0.5 * x
 
 
-fig, axs = plt.subplots(2, 2, sharex=True, sharey=True)
-axs[0, 0].set(xlim=(0, 20), ylim=(2, 14))
+fig, axs = plt.subplots(2, 2, sharex=True, sharey=True, figsize=(6, 5.5),
+                        gridspec_kw={'wspace': 0, 'hspace': 0})
```

Comment:
Not a fan of how this looks:

![Boo](https://user-images.githubusercontent.com/1562854/67166843-0646aa00-f348-11e9-8682-1b75948cf4dd.png)


If you use `constrained_layout=True` its larger and there is white space (which Tufte would like ;-)

![SC](https://user-images.githubusercontent.com/1562854/67166854-368e4880-f348-11e9-9543-0ed54485a54a.png)

Reply:
What I don‘t like is that the horizontal and vertical spacing is different. Can I force the same without switching to AxesGrid? AFAIK only by fine-tuning the figure size.

Reply:
Yes spacing is relative to axes size.  It’s a bit of an unfortunate choice.   You can also fine tune the spacing but that’s not a lot better than messing with the figure size.

Reply:
Interestingly, I seems, I cannot tune the relative whitespace between the Axes by adjusting the figure size when using constrained_layout. Is that to be expected?

Reply:
https://matplotlib.org/tutorials/intermediate/constrainedlayout_guide.html#padding-and-spacing.  

Whats going on here is that the '0' overhangs the x axis and constrained layout makes some space for it.  If you left your ticks pointing outwards, or didn't force 0 to be a tick then this doesn't become a problem.  

![boo2121](https://user-images.githubusercontent.com/1562854/67238553-36528380-f402-11e9-86b0-38227329870d.png)

Reply:
I've refrained from constrained layout because I really want the ticks in and the xticks at the edges and the horizontal and vertical spacing equal.

Reply:
Hey, it’s your example (now) ;-)